packages/fakta-beregning/src/components/fellesFaktaForATFLogSN/VurderFaktaBeregningPanel.jsx

```diff
@@ -244,6 +244,10 @@ export const transformValuesVurderFaktaBeregning = (values, alleBeregningsgrunnl
       const faktaBeregningValues = currentFormValues;
       const beg = faktaBeregningValues[BEGRUNNELSE_FAKTA_TILFELLER_NAME];
       const kode = erOverstyring(currentFormValues) ? OVERSTYRING_AV_BEREGNINGSGRUNNLAG : VURDER_FAKTA_FOR_ATFL_SN;
+      const bg = alleBeregningsgrunnlag[index];
+      const stp = bg.faktaOmBeregning.avklarAktiviteter
```

Comment:
Burde man kanskje kalle den for stpOpptjening for å vise at det er eit anna skjæringstidspunkt enn det som ligger på beregning?

Reply:
Jeg setter den jo lik den av de to vi har. Ikke nødvendigvis opptjening. Tenker du det er bedre å sette den lik kun opptjening å feile om den dto'en ikke er satt?

Reply:
Eg tenkjer jo at vi alltid burde bruke stpOpptjening. Om dto'en ikkje er satt bør vi heller fikse det enn å bruke eit anna skjæringstidspunkt, tenkjer eg.